Bottom: bd672075a41ee0b0ed3c27608592b1f89209a5a4
Top:    df0418b04a3a730965e919a398f28921eb417e2a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-21 23:44:49 +0000

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index b451980..08ffa53 100644
--- a/configure.ac
+++ b/configure.ac
@@ -386,8 +386,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
